socorro/unittest/external/postgresql/test_crash_data.py

```diff
@@ -1,216 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-import os
-
-from configman import ConfigurationManager, Namespace
-from mock import Mock
-import pytest
-
-from socorro.lib import (
-    MissingArgumentError,
-    ResourceNotFound,
-    ResourceUnavailable
-)
-from socorro.unittest.external.postgresql.test_crashstorage import a_processed_crash
-from socorro.external.postgresql import crash_data, crashstorage
-from socorro.unittest.testbase import TestCase
-
-
-class TestIntegrationPostgresCrashData(TestCase):
-
-    def setUp(self):
-        super(TestIntegrationPostgresCrashData, self).setUp()
-        self.config_manager = self._common_config_setup()
-        self._truncate()
-        with self.config_manager.context() as config:
-            store = crashstorage.PostgreSQLCrashStorage(config.database)
-
-        # First we need to create the partitioned tables.
-        connection = store.database.connection()
-        cursor = connection.cursor()
-        table_data = (['reports', '1', '{id,uuid}',
-                       '{date_processed,hangid,"product,version",reason,signature,url}',
-                       '{}', 'date_processed', 'TIMESTAMPTZ'],
-                      ['raw_crashes', '4', '{uuid}', '{}', '{}', 'date_processed',
-                       'TIMESTAMPTZ'],
-                      ['processed_crashes', '6', '{uuid}', '{}', '{}', 'date_processed',
-                       'TIMESTAMPTZ'])
-        query = """
-            INSERT INTO report_partition_info
-            (table_name, build_order, keys, indexes, fkeys, partition_column,
-             timetype)
-            VALUES (%s, %s, %s, %s, %s, %s, %s);
-        """
-        cursor.executemany(query, table_data)
-        connection.commit()
-
-        cursor.execute("SELECT weekly_report_partitions(2, '2012-03-14');")
-        cursor.execute("SELECT weekly_report_partitions(2, '2012-08-20');")
-        connection.commit()
-
-        # A complete crash report (raw, dump and processed)
-        fake_raw_dump_1 = 'peter is a swede'
-        fake_raw_dump_2 = 'lars is a norseman'
-        fake_raw_dump_3 = 'adrian is a frenchman'
```

Comment:
I like reducing the amount of code we have to maintain, but I will miss some of it.

Reply:
:)